inference-engine/tests/functional/plugin/gna/shared_tests_instances/subgraph_tests/concat_quantization.cpp

```diff
@@ -21,7 +21,7 @@ const std::vector<std::map<std::string, std::string>> configs = {
     }
 };
 
-INSTANTIATE_TEST_CASE_P(ConcatQuantization, ConcatQuantization,
+INSTANTIATE_TEST_CASE_P(smoke_ConcatQuantization, ConcatQuantization,
```

Comment:
*smoke* prefix is using as pre-commit gtest filter in old functional tests in CI and will be reused in the future for new functional tests (it is not enabled yet because running of func tests is very fast). I suppose, we shoul not remove `smoke_`

Reply:
@iefode i dint get - we should remove it or not? I though we need to add this to enable tests in precomit.

Reply:
@iefode, please clarify, what should we do for a test to be enabled in precommit? Or do you mean this test should have been running already in precommit?